src/CommonGrids/CommonGrids.jl

```diff
@@ -0,0 +1,647 @@
+"""
+    CommonGrids
+
+CommonGrids contains convenience constructors for common
+grids. Constructors in this module accept integer
+keyword inputs, so they are dynamically created. You may
```

Comment:
What do you mean with "integer keyword inputs"?

Reply:
Meaning `n_quad_points` in

```julia
grid = ExtrudedCubedSphereGrid(;
    z_elem = 10,
    z_min = 0,
    z_max = 1,
    radius = 10,
    h_elem = 10,
    n_quad_points = 4,
)
```
It's passed as an `Int`, so that's all the compiler knows. Which means, `quad = Quadratures.GLL{n_quad_points}()` inside `ExtrudedCubedSphereGrid` is a dynamically constructed type (i.e., a type instability).

It's not a performance issue for us, but if a user did this in a performance-critical section, then it could be for them (I doubt that would be the case, so I think it's okay).

The only way we could make this type stable is if we changed the keyword to positional argument and caught the argument through the type space, for example:

```julia
grid = ExtrudedCubedSphereGrid(n_quad_points; ....)
function ExtrudedCubedSphereGrid(::Type{n_quad_points}; ....) where {n_quad_points}
    ...
end
```
But this isn't really user-friendly in the way I think we want these constructors to be.

Reply:
Thank, I understand now. I think the phrasing is unclear, so I'd maybe add an explanation like this somewhere in the comments (or you could add a subsection in the module docstring).

Reply:
I'll re-phrase/simplify it a bit.